### PR TITLE
Testing To-Do list: Part2

### PR DESCRIPTION
**Pull request by monika-soam**
**Testing**

- Used jest framework for testing.
- Created a test file ([..].test. js) for a file containing the add item and delete the item and updated items, edit items and clear all completed functions that need to be tested.
- Grouped tests using the describe() method.
- No linters and followed the GitHub flow

### CODE REVIEW

File: src/__test__/task.test.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+});
+
+describe('update status', () => {
+  test('update task', () => {
+    const updateList = new TaskList();
+    updateList.addTask('Test');
+    updateList.updateStatus(1);
+    expect(updateList.list[1].completed).toBe(true);
+  });
+});
```
Comment (TedLivist) [inline src/__test__/task.test.js]: - [x] Good job on adding the tests for updating the `completed` attribute of the tasks, however, according to the requirements, you are still missing tests for editing the task `description` and the 'Clear all completed' functionality. Kindly add them.

  lekan-test

File: README.md [hunk 2]
```diff
@@ -102,4 +102,10 @@ All source code files are licensed under the permissive zlib license
 (http://opensource.org/licenses/Zlib) unless marked differently in a particular folder/file.
```
Comment (TedLivist) [inline README.md:108]: - [x] In your README, kindly include instructions on how to run the tests: `npm test`.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @monika-soam and @d2ndjim,

Good job so far!
✔️ Good documentation
✔️ Tests are passing
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__test__/task.test.js (hunk 1 above)
Comment (monika-soam) [reply src/__test__/task.test.js]: Fixed it

File: README.md (hunk 2 above)
Comment (monika-soam) [reply README.md:108]: Fixed it

File: src/__test__/task.test.js (hunk 1 above)
Comment (monika-soam) [reply src/__test__/task.test.js]: Thank you for the review

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @d2ndjim, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**